packages/react-signer/src/PaymentInfo.tsx

```diff
@@ -67,6 +79,9 @@ function PaymentInfo ({ accountId, className = '', extrinsic, isHeader }: Props)
       {isFeeError && (
         <MarkWarning content={t('The account does not have enough free funds (excluding locked/bonded/reserved) available to cover the transaction fees without dropping the balance below the account existential amount.')} />
       )}
+      {isDryRunError && (
+        <MarkError content={t('The transaction would not be successfully executed')} />
```

Comment:
I would ensure we also bring up that it failed via the dry run.

So something like: `The transaction did not successfully pass a dry run, something something`

Reply:
I just changed it to `The transaction did not pass a dry run and would likely not be successfully executed`. Is that OK?